### ISSUE TITLE
a11y: labels, tooltips, ARIA attributes

### ISSUE THREAD

**Issue by candu** [CityofToronto/bdit_flashcrow#636]
**Description**
Many of our buttons, input elements, etc. need proper labels and other non-visual affordances to be accessible.  This task addresses that.

**Acceptance Criteria**
- [x] map legend buttons: "show / hide {{layerName}}";
- [x] delete request comment: "delete comment";
- [x] form elements on `<v-select>` (may need to create `FcSelect` wrapper to properly address this);
- [ ] dropdown menus: items not contained by landmark (may need `role` or other container-identifying attributes);
- [x] multi-location selector:
  - [x] `MULTI` mode: needs labels on readonly inputs (e.g. "Location #{{index}}: {{location}}");
  - [x] `MULTI_EDIT` mode: tooltips on X buttons (e.g. "Remove Location #{{index}}");
- [x] Bulk Request Study: step 1 / 2 need `aria-label` / tooltips on checkboxes to select / deselect locations (e.g. "Remove {{location}} from Request" / "Add {{location}} to Request";
- [x] `FcDataTableRequests`:
  - [x] selection column header should have screen reader-only text, just like "Actions" column;
  - [x] selection checkboxes should have labels / tooltips;
  - [x] "Expand Bulk..." label: should change to "Collapse..." when already open;
- [x] `FcIconLocationMulti`:
  - [x] make sure icon has proper title;
  - [x] make sure text on numbered location icons is ignored by screen readers (to avoid confusion)

**Additional Notes**
Lots of small details here, but each one should be relatively quick.
